deployment/arm/deploy-global.sh

```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
+set -e
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 businessUnit=$1
 appName=$2
 env=$3
+primaryRegion=$4
```

Comment:
This script was initially set to handle 1-n number of regions.  Instead of specifying primary, secondary, etc... can we just check the primary args then validate the length of the argument list for your quick exit check?